function/values.yaml

```diff
@@ -32,4 +32,6 @@ triggerPodIdentityProvider: "azure" # none | azure | gcp | spiffe | aws-eks | aw
 triggers: []
 rollout:
   strategy: gradual # Default strategy deletes the running jobs when the job spec changes
-  propagationPolicy: background
\ No newline at end of file
+  propagationPolicy: background
+scalingStrategy:
```

Comment:
Can u remove the scaling strategy pls, it needs more parameters than just this to define a custom strategy and probably more thought on how it can be used.

Reply:
I wanted to use the `accurate` scaling strategy.

Would it be ok to expose at the top level and only support `accurate` or `default`?

Reply:
I've added this for consideration https://github.com/hmcts/chart-function/pull/34/commits/be3d6c6cda7b74c8259735720607c6b503166013